indico/modules/events/registration/client/js/components/participant-list/ParticipantTable.module.scss

```diff
@@ -0,0 +1,45 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+.cell-img {
```

Comment:
The table is now quite inconsistent when some registrations have pictures and the others don't. In the old version we were using a fixed `64px` height, either on the image or an invisible placeholder div, which avoided this.

![image](https://github.com/user-attachments/assets/6d34f96a-c061-4741-87ea-953250f8fdd8)
![image](https://github.com/user-attachments/assets/7033a950-59a6-46ce-a725-2a3a737f3aa8)

Reply:
Done. Standard 64px height